src/main/java/io/in_toto/keys/RSAKey.java

```diff
@@ -161,17 +153,16 @@ public AsymmetricKeyParameter getPublic() throws IOException {
             return null;
         return PublicKeyFactory.createKey(this.kpr.getPublicKeyInfo());
     }
-
+	
     /**
      * Convenience method to serialize this key as a PEM
      *
      * @param filename the filename to where the key will be written to.
      */
     public void write(String filename) {
         try {
-            FileWriter out = new FileWriter(filename);
-            // XXX: right now we are *not* serializing public keys, although
-            // that should be trivial
+			
+			FileWriter out = new FileWriter(filename);
```

Comment:
this is still using tabs

Reply:
done. 1d38e7918d9a29523da6fa7cafe67b3aaf521c93